autoPyTorch/core/autonet_classes/autonet_image_data.py

```diff
@@ -93,9 +94,10 @@ def _apply_default_pipeline_settings(pipeline):
         net_selector.add_network('resnet152', ResNet152)
         net_selector.add_network('darts', DARTSImageNet)
         net_selector.add_network('mobilenet', MobileNet)
-        net_selector._apply_search_space_update('resnet:nr_main_blocks', [2, 4], log=False)
-        net_selector._apply_search_space_update('resnet:widen_factor_1', [0.5, 8], log=True)
-
+        net_selector.add_network('resnet9', ResNet9)
+        # net_selector._apply_search_space_update('resnet:nr_main_blocks', [2, 4], log=False)
```

Comment:
Should these be here or not?

Reply:
These search space updates cause error when I tried running any network other than resnet. Without these lines, these hyperparameters are still in the resnet search space as they have been added to the get_configspace method of the model.

Reply:
These search space updates cause error when I tried running any network other than resnet. Without these lines, these hyperparameters are still in the resnet search space as they have been added to the get_configspace method of the model.